### PR TITLE
Add Klarna payNowStandalone to settings

### PR DESCRIPTION
**Pull request by lucasxciv**
## Summary

Add Klarna method Pay Now Standalone to settings.

## Checklist

- [X] Writing style
- [X] API design standards

### CODE REVIEW

File: openapi/components/schemas/GatewayAccountConfig/Klarna.yaml [hunk 1]
```diff
@@ -34,3 +34,7 @@ allOf:
               - EU
               - NA
               - OC
+          payNowStandalone:
```
Comment (ashkarpetin) [inline openapi/components/schemas/GatewayAccountConfig/Klarna.yaml:37]:
```suggestion
          usePayNowStandalone:
```

**Review by ashkarpetin — DISMISSED** [review]

File: openapi/components/schemas/GatewayAccountConfig/Klarna.yaml [hunk 2]
```diff
@@ -34,3 +34,7 @@ allOf:
               - EU
               - NA
               - OC
+          usePayNowStandalone:
+            type: boolean
+            description: Specifies if Klarna Pay Now Standalone method must be used.
+            default: false
```
Comment (nanobash) [inline openapi/components/schemas/GatewayAccountConfig/Klarna.yaml:40]: Perhaps?

```suggestion
            description: Specifies whether Klarna pay now standalone method should be used.
            default: false
            example: false
```

**Review by nanobash — DISMISSED** [review]

File: openapi/components/schemas/GatewayAccountConfig/Klarna.yaml (hunk 2 above)
Comment (lucasxciv) [reply openapi/components/schemas/GatewayAccountConfig/Klarna.yaml:40]: Look good! @noon-dawg WDYT?

**Review by i-yasko — DISMISSED** [review]

File: openapi/components/schemas/GatewayAccountConfig/Klarna.yaml (hunk 2 above)
Comment (noon-dawg) [reply openapi/components/schemas/GatewayAccountConfig/Klarna.yaml:40]: I think the addition of 'whether' makes the sentence more readable, but 'should' may sound passive

How about: `Specifies whether the Klarna pay now standalone method must be used.`

Comment (lucasxciv) [reply openapi/components/schemas/GatewayAccountConfig/Klarna.yaml]: Pushed in [e4f96c8](https://github.com/Rebilly/api-definitions/pull/1718/commits/e4f96c8f87ead012bd7928234e0852d8bd1dbf8e). Thanks!

**Review by noon-dawg — APPROVED** [review]
Looks good on the docs side

**Review by ashkarpetin — APPROVED** [review]

**Pull request merged**